hack/verify-all.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# A single script that runs a predefined set of verify-* scripts.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+SILENT=true
+
+declare -r color_red="\033[0;31m"
+declare -r color_green="\033[0;32m"
+declare -r color_norm="\033[0m"
+
+# Excluded checks are always skipped.
+EXCLUDED_CHECKS=(
+  "verify-golint.sh"
+  "verify-jsonformat.sh"
+  "verify-open-ports.sh"
+)
+
+function is-excluded {
+  if [[ $1 -ef ${BASH_SOURCE} ]]; then
+    return
+  fi
+  for e in ${EXCLUDED_CHECKS[@]}; do
+    if [[ $1 -ef "$OS_ROOT/hack/$e" ]]; then
+      return
+    fi
+  done
+  return 1
+}
+
+function run-cmd {
+  if ${SILENT}; then
+    "$@" &> /dev/null
+  else
+    "$@"
+  fi
+}
+
+function run-checks {
+  local -r pattern=$1
+
+  for t in $(ls ${pattern})
+  do
+    if is-excluded "${t}" ; then
+      echo "Skipping ${t}"
+      continue
+    fi
+    echo -e "Verifying ${t}"
+    local start=$(date +%s)
+    run-cmd bash "${t}" && tr=$? || tr=$?
+    local elapsed=$(($(date +%s) - ${start}))
+    if [[ ${tr} -eq 0 ]]; then
+      echo -e "${color_green}SUCCESS${color_norm}  ${t}\t${elapsed}s"
+    else
+      echo -e "${color_red}FAILED${color_norm}   ${t}\t${elapsed}s"
+      ret=1
+    fi
+  done
+}
+
+while getopts ":v" opt; do
```

Comment:
We don't have any precedents for this, verbosity in our scripts is always `VERBOSE= true`.